anise/src/almanac/transform.rs

```diff
@@ -178,4 +181,36 @@ impl Almanac {
                 action: "transform provided state",
             })
     }
+
+    /// Returns the unitary 3D vector between two [Frame]s (solid bodies) at desired [Epoch]
+    pub fn unit_vector(
+        &self,
+        target_frame: Frame,
+        observer_frame: Frame,
+        epoch: Epoch,
+        ab_corr: Option<Aberration>,
+    ) -> AlmanacResult<Vector3> {
+        let state = self.transform(target_frame, observer_frame, epoch, ab_corr)?;
+        let normalized = state.radius_km / state.rmag_km();
```

Comment:
```suggestion
        Ok(state.radius_km / state.rmag_km())
```

The operation returns a `Vector3`, so this should work!